content/en-US/api/IdleDetector.md

```diff
@@ -0,0 +1,76 @@
+---
+recipe: api-interface
+title: 'IdleDetector'
+mdn_url: /en-US/docs/Web/API/IdleDetector
+specifications: https://wicg.github.io/idle-detection/#api-idledetector
+browser_compatibility: api.IdleDetector
+---
+
+**When this feature ships, the content below will live on MDN under
+[developer.mozilla.org/en-US/docs/Web/API](https://developer.mozilla.org/en-US/docs/Web/API).**
+
+## Description
+
+The `IdleDetector` interface of the Idle Detection API provides events
+indicating when the user is no longer interacting with their device or the
+screen has locked.
+
+This interface requires a secure context.
+
+## Constructor
+
+Creates a new `IdleDetector` object.
+
+## Properties
+
+**`IdleDetector.userState`**
+
+Returns either `"active"` to indicate that the user has interacted with the
+device within the threshold provided to `start()` or `"idle"` if they have not.
+This attribute will return `null` before `start()` is called.
+
+**`IdleDetector.screenState`**
+
+Returns either `"locked"` if the device's screen is locked or `"unlocked"` if it
+is not. This attribute will return `null` before `start()` is called.
+
+## Events
+
+**`IdleDetector.onchange`**
+
+Called when the value of `userState` or `screenState` has changed.
```

Comment:
What event type is passed to this callback?

Reply:
This event uses the generic `Event` type. How should I phrase that here?

Reply:
My answer is in the adjacent suggestion.